src/main.rs

```diff
@@ -0,0 +1,57 @@
+use rand::Rng;
+use std::fmt;
+
+fn main() {
+    let mut bag = TileBag::new_default();
+    for i in 0..1000 {
+        let tile = bag.draw_tile();
+        println!("{}", tile)
+    }
+}
+
+struct TileBag {
+    bag: Vec<char>,
+    rng: rand::rngs::ThreadRng,
+}
+
+impl TileBag {
+    fn new(letter_distribution: [u8; 26]) -> Self {
+        let mut bag: Vec<char> = vec![];
+        let range: std::ops::Range<u8> = std::ops::Range { start: 0, end: 26 };
+        for letter in range {
+            for i in 0..letter_distribution[letter as usize] {
+                bag.push((letter + 65) as char);
+            }
+        }
+
+        TileBag {
+            bag: bag,
+            rng: rand::thread_rng(),
+        }
+    }
+
+    fn new_default() -> Self {
+        Self::new([
+            // banagrams letter distribution
+            13, 3, 3, 6, 18, 3, 4, 3, 12, 2, 2, 5, 3, 8, 11, 3, 2, 9, 6, 9, 6, 3, 3, 2, 3, 2,
+        ])
+    }
```

Comment:
Here you should look at implementing the `Default` trait, since this is a default function that needs no arguments.

```rust
impl Default for TileBag {
  fn default() -> Self {
    Self::new([
      // banagrams letter distribution
      13, 3, 3, 6, 18, 3, 4, 3, 12, 2, 2, 5, 3, 8, 11, 3, 2, 9, 6, 9, 6, 3, 3, 2, 3, 2,
    ])
  }
}
```

And then `TileBag::default()`.

More idiomatic, and also integrates with a bunch of std helper functions. i.e. if you now had the type `Option<TileBag>` you can call `.unwrap_or_default()` rather than handling that logic yourself